permissions/pom-build-monitor.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "build-monitor"
+github: "jan-molak/jenkins-build-monitor-plugin"
+paths:
+- "org/jenkins-ci/plugins/build-monitor"
```

Comment:
Should also probably remove this path from `permissions/plugin-build-monitor-plugin.yml` as it was added there in error.